docs/pages/moving_forward.rst

```diff
@@ -1,6 +1,13 @@
 Moving Forward
 ==============
 
+Cozmo Limitations
+-----------------
+
+During the course of development on this project, we have noticed a few limitations that can affect future additions.
```

Comment:
Can you add a blank line after this? That should fix the entire intro paragraph from being bolded.

Reply:
Also you cold **bold** the `Three cube limitation:` and `Lighting:` by wrapping them in asterisks `*`.